### ISSUE TITLE
 confusing definition of the var[x] in BatchNorm2D document

### ISSUE THREAD

**Issue by wangyongjie-ntu** [pytorch/pytorch#39330]
## 📚 Documentation

https://pytorch.org/docs/master/generated/torch.nn.BatchNorm2d.html

Here the var[x] of the denominator is population variance dividing by N. It should be precisely clarified here. Otherwise, the reader may use the sample variance by calling tensor.var() which is divided by N - 1.
